docs/systemd.md

```diff
@@ -58,11 +58,22 @@ dinamically, you can find the remaining options needed in
 
 ## Systemd units on WSL
 
-Unfortunately, `systemd` is not officially supported on WSL.
+`systemd` is supported on WSL starting from version `0.67.6` and higher. Note that you need to have Windows 11 to install this version.
```

Comment:
Perhaps it would be good to also provide instructions on how to check which version of WSL one has installed here